### PR TITLE
Button: `view_plain` modifier described

### LINKED ISSUE #1454: button: add to docs information about 'view_plain' mod

### PR DESCRIPTION
**Pull request by innabelaya**
Closes #1454

### CODE REVIEW

File: common.blocks/button/button.en.md [hunk 1]
```diff
@@ -293,6 +293,29 @@ The modifier changes visual representation of the button. For example, use it if
 }
 ```
 
+##### Boarderless button (`view` modifier with `plain` value)
```
Comment (sipayRT) [inline common.blocks/button/button.en.md]: Borderless?

File: common.blocks/button/button.en.md [hunk 2]
```diff
@@ -293,6 +293,29 @@ The modifier changes visual representation of the button. For example, use it if
...
+
+Use this modifier to create a button based on the other block. For example, iconed-button ([icon](../icon/icon.en.md)).
+
+```js
+{
+    block : 'button',
+    mods : { theme : 'islands', size : 'm', view : 'plain' },
+    text : 'Boarderless button'
```
Comment (sipayRT) [inline common.blocks/button/button.en.md]: and here

File: common.blocks/button/button.en.md (hunk 1 above)
Comment (innabelaya) [inline common.blocks/button/button.en.md]: thanks!

**Comment by sipayRT** [comment]
:+1:

**Pull request merged**